projects/1.20/assets/pams-harvestcraft-2-food-core/pamhc2foodcore/lang/zh_cn.json

```
@@ -0,0 +1,150 @@
+  "item.pamhc2foodcore.groundfishitem": "鱼肉碎",
+  "item.pamhc2foodcore.groundmuttonitem": "羊肉碎",
+  "item.pamhc2foodcore.groundporkitem": "猪肉碎",
+  "item.pamhc2foodcore.groundrabbititem": "兔肉碎",
+  "item.pamhc2foodcore.gummybearsitem": "小熊软糖",
+  "item.pamhc2foodcore.hotchocolateitem": "热可可",
+  "item.pamhc2foodcore.hotdogitem": "热狗",
+  "item.pamhc2foodcore.icecreamitem": "冰激凌",
+  "item.pamhc2foodcore.jellybeansitem": "糖豆",
+  "item.pamhc2foodcore.jellydonutitem": "果酱甜甜圈",
+  "item.pamhc2foodcore.juiceritem": "榨汁机",
+  "item.pamhc2foodcore.macncheeseitem": "奶酪通心粉",
+  "item.pamhc2foodcore.marshmellowchicksitem": "小鸡棉花糖",
+  "item.pamhc2foodcore.marshmellowsitem": "棉花软糖",
+  "item.pamhc2foodcore.mashedpotatoesitem": "马铃薯泥",
+  "item.pamhc2foodcore.mayonaiseitem": "蛋黄酱",
+  "item.pamhc2foodcore.meatloafitem": "烘肉卷",
+  "item.pamhc2foodcore.melonjellyitem": "西瓜酱",
+  "item.pamhc2foodcore.melonjellytoastitem": "西瓜酱吐司",
+  "item.pamhc2foodcore.melonjuiceitem": "西瓜汁",
+  "item.pamhc2foodcore.melonsmoothieitem": "西瓜沙冰",
+  "item.pamhc2foodcore.mixingbowlitem": "搅拌钵",
+  "item.pamhc2foodcore.muttonjerkyitem": "羊肉干",
+  "item.pamhc2foodcore.noodlesoupitem": "面条汤",
+  "item.pamhc2foodcore.p8juiceitem": "P8果汁",
+  "item.pamhc2foodcore.pastaitem": "通心粉",
+  "item.pamhc2foodcore.pickledbeetsitem": "腌甜菜",
+  "item.pamhc2foodcore.porkjerkyitem": "猪肉干",
+  "item.pamhc2foodcore.potatochipsitem": "薯片",
+  "item.pamhc2foodcore.potatosoupitem": "马铃薯汤",
+  "item.pamhc2foodcore.potitem": "锅",
+  "item.pamhc2foodcore.potroastitem": "红烧牛肉",
+  "item.pamhc2foodcore.pumpkinbreaditem": "南瓜面包",
+  "item.pamhc2foodcore.pumpkincheesecakeitem": "南瓜奶酪蛋糕",
+  "item.pamhc2foodcore.pumpkincupcakeitem": "南瓜纸杯蛋糕",
+  "item.pamhc2foodcore.pumpkinmuffinitem": "南瓜松饼",
+  "item.pamhc2foodcore.pumpkinsoupitem": "南瓜汤",
+  "item.pamhc2foodcore.pumpkinyogurtitem": "南瓜酸奶",
+  "item.pamhc2foodcore.rabbitjerkyitem": "兔肉干",
+  "item.pamhc2foodcore.roastedsunflowerseedsitem": "烤葵花籽",
+  "item.pamhc2foodcore.rolleritem": "擀面杖",
+  "item.pamhc2foodcore.saltitem": "盐",
+  "item.pamhc2foodcore.saucepanitem": "深平底锅",
+  "item.pamhc2foodcore.scrambledeggitem": "西式炒蛋",
+  "item.pamhc2foodcore.skilletitem": "平底锅",
+  "item.pamhc2foodcore.smoresitem": "棉花糖巧克力夹心饼干",
+  "item.pamhc2foodcore.softpretzelitem": "椒盐脆饼",
+  "item.pamhc2foodcore.stewitem": "大锅炖",
+  "item.pamhc2foodcore.stockitem": "底汤",
+  "item.pamhc2foodcore.sunflowerseedsitem": "葵花籽",
+  "item.pamhc2foodcore.sweetberryjellyitem": "甜浆果酱",
+  "item.pamhc2foodcore.sweetberryjellytoastitem": "甜浆果酱吐司",
+  "item.pamhc2foodcore.sweetberryjuiceitem": "甜浆果汁",
+  "item.pamhc2foodcore.sweetberrypieitem": "甜浆果派",
+  "item.pamhc2foodcore.sweetberrysmoothieitem": "甜浆果沙冰",
+  "item.pamhc2foodcore.sweetberryyogurtitem": "甜浆果酸奶",
+  "item.pamhc2foodcore.toastitem": "吐司",
+  "item.pamhc2foodcore.trailmixitem": "什锦蜜饯",
+  "item.pamhc2foodcore.vinegaritem": "醋",
+  "item.pamhc2foodcore.yogurtitem": "酸奶",
+  "itemGroup.pamhc2foodcore": "潘马斯农场2|食物核心"
```

```suggestion
  "itemGroup.pamhc2foodcore": "潘马斯农场2丨食物核心"
```
创造模式标签页的分隔符应采用汉字`丨`
请全文通查